toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1489,11 +1489,12 @@ func ProvisionUserSSHCerts(installChroot *safechroot.Chroot, username string, ss
 
 	// Skip user SSH directory generation when not provided with public keys
 	// Let SSH handle the creation of this folder on its first use
-	if len(sshPubKeyPaths) == 0 {
-		return
-	}
+	if len(sshPubKeyPaths) == 0 && len(sshPubKeys) == 0 {
+        return nil
```

Comment:
Be careful with changing formatting/whitespace. It isn’t a bad idea to use `go fmt` before committing. If you use VSCode, it will auto format whenever you save a file.